src/components/Image/Image.js

```diff
@@ -24,6 +24,10 @@ import type { FastImageProps, FastImageSource } from 'react-native-fast-image';
 
 export type ImageProps = FastImageProps;
 
+type Props = {
+  fallbackSource?: string,
+}
+
```

Comment:
The idiomatic way would be to extend ImageProps with `fallbackSource` prop:

```suggestion
export type ImageProps = {|
  ...FastImageProps,
  fallbackSource?: ImageSource,
|};
```
ImageSource should be imported from utils/types-react-native.